utils/trivy/trivy.yaml

```diff
@@ -16,7 +19,7 @@ db:
   no-progress: false
   repository: ghcr.io/aquasecurity/trivy-db
   skip-update: false
-debug: false
+debug: true
```

Comment:
Are we going to land with this for the time being or is this just for this PR?

Reply:
Yes, let's keep debug enabled until we see that the solution is stable.